keywording/text.xml

```diff
@@ -237,23 +242,57 @@ to do this.
-the arch teams are happy for help, so long as they know what is going on.
+Stabilization, i.e., moving an ebuild from <c>~arch</c> ("testing") to
+<c>arch</c> ("stable"), is done by the relevant architecture teams. If you have
+access to exotic hardware but are not on the arch teams, you may wish to make
+individual arrangements <d/> the arch teams are happy for help, so long as they
+know what is going on.
 </p>
 
 <p>
 In order to request stabilization of an ebuild, file a bug to the package's
 maintainer (which may be yourself) in the Stabilization component, and list
 all secondary maintainers in the bug's CC. When the maintainers consider the
 ebuild to be ready for stabilization, they will add the relevant architecture
-teams to the CC list. They can do it manually, or they can fill the package
+teams to the CC list. They can fill the package
 list field, add the <c>CC-ARCHES</c> keyword, and let
 <uri link="https://dev.gentoo.org/~mgorny/doc/nattka/">NATTkA</uri>
-automatically add arch teams to CC.
-That way teams can remove themselves from the list when they are done, giving
-a clear indication of which teams still have to stabilize a package.
+automatically add arch teams to CC (preferred), or do this manually.
+That way, teams can remove themselves from the list when they are done, giving
+a clear indication of which teams still remain to stabilize a package.
+</p>
+
+<p>
+It is <b>strongly</b>
+<uri link="https://archives.gentoo.org/gentoo-dev/message/cd62f6be924f6a0f76b68a07d33b256a">
+recommended</uri> that <c>NATTkA</c> be used to file keywording
+or stabilization bugs to ensure that arches are not accidentally forgot about;
+the automation removes an often error-prone step in the arch-testing process. Ask
+in #gentoo-dev or on the gentoo-dev mailing list if you need help.
+</p>
+
+<p>
+<c>NATTkA</c> will set the <c>sanity-check</c> field on Bugzilla to either
+<c>-</c> or <c>+</c> depending on whether the package list results in a consistent
+dependency graph. You may need to add more packages or adjust the archs listed
+for each package based on the output the bot posts as a comment on the bug. Please
+note that arch teams may not process or notice bugs with a blank or <c>-</c>
+sanity-check, so please fix this if it occurs <d/> or ask for help to do so.
+</p>
+
+<note>
+Note that sometimes NATTkA may do <e>nothing</e> if it has concluded the package
+list is complete. You may wish to run <c>nattka sanity-check BUG</c> where
+<c>BUG</c> is the bug number you're having an issue with in order to debug the
+issue to receive verbose output.
+</note>
+
+<p>You may wish to read NATTkA's
+<uri link="https://github.com/mgorny/nattka#filing-keywordingstabilization-bugs">
+documentation</uri> for help on package list syntax or other information about
+the tool. The documentation details syntax which can be used, such as '*' to
+represent 'all previously stable arches', or '^' to copy the archs from the
```

Comment:
```suggestion
the tool. The documentation details syntax which can be used, such as <c>*</c> to
represent "all previously stable arches", or <c>^</c> to copy the archs from the
```
(`<c>` instead of quotes, double quotes instead of single quotes.)